internal/services/kusto/kusto_cluster_resource.go

```diff
@@ -409,6 +433,10 @@ func resourceKustoClusterCreate(d *pluginsdk.ResourceData, meta interface{}) err
 		clusterProperties.AllowedIPRangeList = expandKustoListString(v.([]interface{}))
 	}
 
+	if v, ok := d.GetOk("callout_policy"); ok {
```

Comment:
Arguably this check is redundent, whilst let's leave it as is to keep consistency.